homeassistant/components/axis/config_flow.py

```diff
@@ -156,7 +156,7 @@ async def async_step_dhcp(self, discovery_info: dict):
         return await self._process_discovered_device(
             {
                 CONF_HOST: discovery_info[IP_ADDRESS],
-                CONF_MAC: format_mac(discovery_info.get(MAC_ADDRESS)),
+                CONF_MAC: format_mac(str(discovery_info.get(MAC_ADDRESS))),
```

Comment:
This basically casts `None` to a string? That seems incorrect.

Reply:
well it is actually not incorrect, but a rather bad solution. Using the default in the get function is a lot better.